mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -735,11 +792,11 @@ is set to true.`,
 							Type:             schema.TypeString,
 							Optional:         true,
 							DiffSuppressFunc: timestampDiffSuppress(time.RFC3339Nano),
-							Description:  `The timestamp of the point in time that should be restored.`,
+							Description:      `The timestamp of the point in time that should be restored.`,
 						},
 						"allocated_ip_range": {
-							Type:             schema.TypeString,
-							Optional:         true,
+							Type:         schema.TypeString,
+							Required:    	true,
```

Comment:
we can not change a field from optional to required in a minor release. It's considered a breaking change and we can only make such changes in major release.